dev/com.ibm.websphere.appserver.jsfApiStub-2.2/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply plugin: "net.ossindex.audit"
```

Comment:
Do we need this plugin?

Reply:
`ossindex` isn't needed here - this was copied from a different project.  I'll remove it.